src/launches/launches.ts

```diff
@@ -90,11 +91,22 @@ export class Launches {
   }
 
   gotoEtherscan(launch: ILaunch, event: Event): boolean {
-    Utils.goto(this.ethereumService.getEtherscanLink(launch.address));
+    if (launch.launchType === LaunchType.LBP ){
+      Utils.goto(this.ethereumService.getEtherscanLink(this.getLbpAddress(launch.address)));
```

Comment:
should just be:  `Utils.goto(this.ethereumService.getEtherscanLink((launch as LbpManager).lbp.address));

(or some type cast like that)

Reply:
implemented your suggestion